spec/app/domain/authentication/authn_k8s/validate_pod_request_spec.rb

```diff
@@ -96,16 +97,153 @@
         )
       end
 
-      it 'raises an error when application identity validation fails' do
-        allow(validate_application_identity).to receive(:call)
-                                                  .and_raise(validate_application_identity_error)
+      it 'raises an error when resource restrictions validation fails' do
+        allow(validate_resource_restrictions).to receive(:call)
+                                                   .and_raise('FAKE_RESOURCE_RESTRICTIONS_ERROR')
 
         expect { subject }.to(
           raise_error(
-            validate_application_identity_error
+            /FAKE_RESOURCE_RESTRICTIONS_ERROR/
           )
         )
       end
+
+      #context "with different annotations for container name" do
```

Comment:
I assume this is the test you have left?

Reply:
yes, implemented a new class - `extract_container_name` and moves these UTS THERE